src/components/__tests__/search.test.tsx

```diff
@@ -22,6 +22,15 @@ const result = [
   },
 ];
 
+const mockGeolocation = {
+  getCurrentPosition: vi.fn(),
+};
+
+Object.defineProperty(global.navigator, 'geolocation', {
+  writable: true,
+  value: mockGeolocation,
```

Comment:
Can this be mocked globally in this file?

Reply:
Yes, but could be nice to do some proper cleaning afterwords (e.g. resetting it)

Reply:
Setting it in `beforeAll` and resetting it in `afterAll` now.